### PR TITLE
Minor tidy ups

### PR DESCRIPTION
**Pull request by lofidevops**
Fixes some typos, removes unused status blocks, adds basic development instructions.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,7 +5,7 @@ env/.done: requirements-dev.txt setup.py
 	env/bin/pip uninstall -y argparse  # we want to use the built-in version of argparse
 	touch $@
 
-.PHONY: text
+.PHONY: test
```
Comment (Roadmaster) [inline Makefile:8]: phony text lol

**Review by Roadmaster — APPROVED** [review]
LGTM

File: Makefile (hunk 1 above)
Comment (maxiberta) [reply Makefile:8]: :clap:

**Review by maxiberta — APPROVED** [review]
+1

**Pull request merged**